src/Sylius/Behat/Context/Setup/ProductAssociationContext.php

```diff
@@ -81,6 +111,29 @@ private function createProductAssociationType($name, $code = null)
     }
 
     /**
+     * @param ProductInterface $product
+     * @param AssociationTypeInterface $productAssociationType
+     * @param ProductInterface $firstProduct
+     * @param ProductInterface|null $secondProduct
+     */
+    private function createProductAssociation(
+        ProductInterface $product,
+        AssociationTypeInterface $productAssociationType,
+        ProductInterface $firstProduct,
+        ProductInterface $secondProduct = null
```

Comment:
Instead of nullable value as a second parameter we can use variadics and then use a foreach.